components/TabbedView/TabbedView.jsx

```diff
@@ -14,6 +20,32 @@ const Navbar = styled.nav.attrs({
   flex-shrink: 1;
   margin: 0 0 25px 0;
```

Comment:
how about of put a property here from spacing (theme)?

Reply:
Done